nucypher/cli/options.py

```diff
@@ -54,9 +54,12 @@
 option_hw_wallet = click.option('--hw-wallet/--no-hw-wallet')
 option_light = click.option('--light', help="Indicate that node is light", is_flag=True, default=None)
 option_lonely = click.option('--lonely', help="Do not connect to seednodes", is_flag=True)
-option_min_stake = click.option('--min-stake', help="The minimum stake the teacher must have to be locally accepted.", type=STAKED_TOKENS_RANGE, default=MIN_ALLOWED_LOCKED_TOKENS)
+option_min_stake = click.option('--min-stake', help="The minimum stake the teacher must have to be locally accepted.", type=STAKED_TOKENS_RANGE, default=MIN_AUTHORIZATION)
 option_parameters = click.option('--parameters', help="Filepath to a JSON file containing additional parameters", type=EXISTING_READABLE_FILE)
 option_participant_address = click.option('--participant-address', help="Participant's checksum address.", type=EIP55_CHECKSUM_ADDRESS)
+option_payment_provider = click.option('--payment-provider', help="Connection URL for payment method", type=click.STRING, required=False)
+option_payment_network = click.option('--payment-network', help="Payment network name", type=click.STRING, required=False)  # TODO: Choices
```

Comment:
In my opinion "payment network" is too specific.
Is that really all this is about?

Reply:
I should clarify.  My comment was really more about ensuring that the correct value is specified from the CLI, instead of having to check it in code. Currently, the type is a string, but I believe the only two values it can be is currently "mumbai" or "polygon".

To clarify, it doesn't have to be defined as a type, we could do something like:
```python
type=click.Choice([NetworksInventory.POLYGON, NetworksInventory.MUMBAI])
```

since they are defined in the NetworksInventory class.

Reply:
ooh that's fancy...

Reply:
I'd like to address this in a follow up 🤞🏻